src/routes/controllers/productsControllers/getProducts.js

```diff
@@ -0,0 +1,12 @@
+import ProductModel from '../../../models/product';
+
+const getProducts = async (req, res) => {
+  try {
+    const products = await ProductModel.find();
```

Comment:
add a condition that will do `res.status(404).json({msg: 'No products found'})` if products.length === 0